galahad/client/formats.py

```diff
@@ -73,3 +74,47 @@ def build_span_classification_document(
 
     document = Document(text=text, annotations=annotations.to_dict(), version=version)
     return document
+
+
+def build_span_classification_response(
+    original_doc: Document, spans: List[List[Span]] = None, version: int = 0
+) -> Document:
+
+    annotated_doc = copy.deepcopy(original_doc)
+
+    assert "t.token" in original_doc.annotations
```

Comment:
We have constants for these in classifier/AnnotationTypes (not the best place, we can move it later, but these should be used).